src/main/java/de/ebf/security/PermissionsConfig.java

```
@@ -61,10 +79,40 @@ public InterfaceBeanScanner protectedResourceInterfaceBeanScanner() {
         return interfaceBeanScanner;
     }
 
+    @Bean
+    public PermissionScanner permissionScanner() {
+        return new DefaultPermissionScanner();
+    }
+
+    @Bean
+    public PermissionModelDefinition permissionModelDefinition(@Autowired PermissionModelFinder permissionModelFinder) {
```

You can throw here a `FatalBeanException` instead of returning null. This is a required bean after all